Add code-oriented questions in the first Python workout (#2681)
enkidevs/curriculum: python/python-core/meet-python/what-is-python.md

## Practice

> Let's write some Python code! Did you remember how to print a message?

Author: ```suggestion
Let's write some Python code! Do you remember how to print a message?
```

> Don't worry, this is just to get you comfortable with code. 
> You'll learn all about how this works in the upcoming lessons

```python
my_age = 23
legal_age = 21
if my_age >= legal_age:
    ???("You can have a beer!")
```

- print
- say
- yell
- log

---

## Revision

> Python is a programming language that can be used for just about anything.

Reviewer: let's switch to the active voice here
```suggestion
Python is a programming language that you can use for just about anything.
```

> We call these kinds of languages ??? languages.

Author: ```suggestion
We call these kinds of languages ???.
```

## Footnotes

[1:Syntax]
The syntax of a programming language sets the rules on how a language can be written and understood by the computer.

The same idea exists in human languages. 

> Just like how English has special types of words like *verb* and *nouns*, as well as rules on which should come first to make a valid sentence, Python has a different set of rules on how to make valid Python statements.

Reviewer: we should remove the "Just like" at the start of the sentence. how does this sound?
```suggestion
The English language has unique types of words like *verb* and *nouns* and rules on which should come first to make a correct sentence; Python has a different set of rules on making valid Python statements.
```
